### PR TITLE
docs: update readme

### PR DESCRIPTION
**Pull request by JingwenBai**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,5 @@
 [![CI test on push](https://github.com/lslzl3000/testci/actions/workflows/ci.yml/badge.svg?branch=main)](https://github.com/lslzl3000/testci/actions/workflows/ci.yml)
 
 test github actions
+
+testbai
```
Comment (lslzl3000) [inline README.md:5]: why?

**Review by lslzl3000 — COMMENTED** [review]
askdjflksjd;fj

File: README.md (hunk 1 above)
Comment (JingwenBai) [reply README.md:5]: just do it!!!

**Review by lslzl3000 — APPROVED** [review]
dasdf

**Comment by lslzl3000** [comment]
related to #7

**Pull request merged**